client/src/components/Modal/Modal.stories.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Modal from './Modal';
+import "../../App.css";
+
+function handleConfirm(evt) {
+  evt.preventDefault()
+  alert('Confirmed!')
+}
+function handleJoin(evt) {
+  evt.preventDefault()
+  alert('Joined!')
+}
+function handleCancel (evt){
+  evt.preventDefault()
+  alert('Canceled!')
+}
+function handleClose (evt){
+  evt.preventDefault()
+  alert('Modal Closed!')
+}
+
+storiesOf('Modal', module)
```

Comment:
Can you work through and create a story for having the modal pop up and close out? Like just a button or link that we can click that triggers the open and then with close functions

Reply:
Hey @ZakkMan from what I can tell, the `FunctionalModal` component is the one that demonstrates functionality as it includes a button and the Modal component itself, and it is where state lives so that showModal prop can be toggle true/false